apiserver/params/crossmodel.go

```diff
@@ -564,6 +564,7 @@ const (
 // needed to make a connection to an external controller.
 type ExternalControllerInfo struct {
 	ControllerTag string   `json:"controller-tag"`
+	Alias         string   `json:"controller-alias"`
```

Comment:
Always there or omitempty is needed?

Reply:
will always be there since it comes from the controller name on the user's client machine